slides/sysdev-shopping-list-stm32mp2/sysdev-shopping-list-stm32mp2.tex

```diff
@@ -0,0 +1,27 @@
+\begin{frame}
+\frametitle{Shopping list: hardware for this course}
+  \begin{columns}
+    \column{0.75\textwidth}
+    \footnotesize
+    \begin{itemize}
+      \item STMicroelectronics STM32MP257f-DK Discovery kit -
```

Comment:
Capital f. Actually writing "STM32MP257f-DK Discovery kit" is a bit redundant as the DK in STM32MP257f-DK means Discovery Kit.